var/spack/repos/builtin/packages/netcdf-cxx/package.py

```diff
@@ -19,9 +19,18 @@ class NetcdfCxx(AutotoolsPackage):
 
     depends_on('netcdf')
 
+    variant(
+        'netcdf4', default=True, description='Compile with netCDF4 support')
+
```

Comment:
Nitpick question: Should this variant be called 'nc4' or 'netcdf4'?  (I'm not sure if I have strong feelings on the answer, just think the question should be asked).

Reply:
I'd say 'netcdf4', to match conventions. Many packages have a 'netcdf' variant, and netCDF itself has a 'parallel-netcdf' variant.